pkg/controller/flow/reconcile.go

```diff
@@ -222,6 +224,11 @@ func (r *reconciler) reconcileChannel(flow *v1alpha1.Flow) (*channelsv1alpha1.Ch
 }
 
 func (r *reconciler) createChannel(flow *v1alpha1.Flow) (*channelsv1alpha1.Channel, error) {
+	clusterBusName, err := r.getDefaultClusterBusName()
```

Comment:
We should add an issue that we should make it possible for a flow to use a Bus or a ClusterBus

Reply:
Yep, I'll create that issue.

Reply:
Created https://github.com/knative/eventing/issues/336